ensure-is-media.js

```diff
@@ -9,16 +9,57 @@ let makeUseIsMedia = (
 // improving the algorithms inside, etc, see this:
 // https://github.com/viewstools/morph/blob/master/ensure-is-media.js
 
-import { useMedia } from 'use-media';
+import { useState, useEffect } from 'react'
+
+const MOCK_MEDIA_QUERY_LIST = {
```

Comment:
a couple of stylistic changes but could we:
- refactor `const` into `let`
- refactor the function definitions to use `function` instead of a variable assignment?